tsl/src/nodes/vector_agg/exec.c

```diff
@@ -339,6 +413,18 @@ compressed_batch_init_vector_quals(VectorAggState *agg_state, VectorAggDef *agg_
 	return &agg_state->vqual_state.vqstate;
 }
 
+/*
+ * Initialize FILTER vector quals for an arrow tuple slot.
+ *
+ * Used to implement vectorized aggregate function filter clause.
+ */
+static VectorQualState *
+arrow_init_vector_quals(VectorAggState *agg_state, VectorAggDef *agg_def, TupleTableSlot *slot)
+{
+	vector_qual_state_init(&agg_state->vqual_state.vqstate, agg_def->filter_clauses, slot);
+	return &agg_state->vqual_state.vqstate;
```

Comment:
Suspicious that this is not covered, do we have a test for aggregate FILTER clause over tam?

Reply:
Yeah, I thought I had tests for this. What would a typical query look like to cover this case?

Reply:
Something like this and other queries in this test https://github.com/timescale/timescaledb/blob/main/tsl/test/sql/vector_agg_filter.sql#L127

Reply:
Strange, I have such a query in a test and if I print something in this function I see it output in the test. It looks like CodeCov didn't get this right....?

Reply:
Seems to have gone away now.